src/System.Private.CoreLib/src/System.Private.CoreLib.csproj

```diff
@@ -941,6 +930,19 @@
       <Link>Interop\Windows\BCrypt\Interop.BCryptGenRandom.cs</Link>
     </Compile>
   </ItemGroup>
+  <ItemGroup Condition="('$(TargetsWindows)'=='true')  And '$(EnableDummyGlobalizationImplementation)' != 'true'">
+    <Compile Condition="'$(EnableWinRT)' != 'true'" Include="System\Globalization\HijriCalendar.Win32.cs" />
```

Comment:
Not sure what you mean here - there are indeed HijriCalendar.Dummy.cs and JapaneseCalendar.Dummy.cs, hence we *only* want to compile HijriCalender.Win(32|RT).cs and JapaneseCalendar.Win(32|RT).cs if EnableDummyGlobalizationImplementation is off. No?